R/dictionary.R

```diff
@@ -96,7 +97,8 @@ reset_rownames <- function(x) {
 #'   * `vec_duplicate_any()`: a logical vector of length 1.
 #'   * `vec_duplicate_all()`: a logical vector of length 1.
 #'   * `vec_duplicate_detect()`: a logical vector the same length as `x`.
```

Comment:
With the new subscript nomenclature I wonder if this should be `vec_duplicate_ind()`, for "indicator". This has the nice relationship that the indicator variant is the `which()` of the location variant. Also the suffix has three letters.

Reply:
After discussion, I've soft deprecated `vec_duplicate_detect()` in favor of `vec_duplicate_flg(x, first = TRUE)`.